broker-daemon/state-machines/fill-state-machine.js

```diff
@@ -140,7 +156,7 @@ const FillStateMachine = StateMachine.factory({
    * @returns {Object} Data to attach to the state machine
    */
   data: function ({ store, logger, relayer, engines }) {
-    return { store, logger, relayer, engines, fill: {} }
+    return { store, logger, relayer, engines, fill: {}, dates: {} }
```

Comment:
would be nice if this could be initialized elsewhere rather than by the implementing state machine.

Reply:
Do you mean by the plugin?